front/src/modules/ui/input/relation-picker/components/CreateNewButton.tsx

```diff
@@ -0,0 +1,14 @@
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+
+import { MenuItem } from '@/ui/menu-item/components/MenuItem';
+
+const StyledCreateNewButton = styled(MenuItem)<{ hovered: boolean }>`
+  ${({ hovered, theme }) =>
+    hovered &&
+    css`
+      background: ${theme.background.transparent.light};
+    `}
+`;
+
+export const CreateNewButton = StyledCreateNewButton;
```

Comment:
@G3root exporting it as CreateNewButton to hide the fact that this is only a Styled component. The outside world shouldn't care about it!